lib/util/string-functions.js

```diff
@@ -116,8 +116,7 @@ exports.wordMatch = function wordMatch (string1, string2) {
 }
 
 exports.removePageNumber = (pageNum, string) => {
-  // eslint-disable-next-line eqeqeq
-  if (pageNum == string) {
+  if (pageNum === string) {
```

Comment:
but the argument isn't a string anymore?

Reply:
In util/pdf:  
text = removePageNumber(j, Number(item.str))

This will coerce the string to a number. Granted the parameter name should be changed.

Reply:
Change the argument name please.